File: src/main/java/gestorAplicacion/comunicacion/ServicioEmail.java
package gestorAplicacion.comunicacion;

/**
 * Implementación de un servicio de envío específico usando EMAIL.
 *
 * @author Juan Daniel Bula, Leidy Daniela Alzate, Oscar Andres Usuga
 */
public class ServicioEmail extends ServicioEnvio {

    /**
     * Constructor asignando el mensaje al email.
     * @param mensaje Mensaje a ser enviado.
     */
    public ServicioEmail(String mensaje) {
        super(mensaje);
    }

    @Override
    public void enviar() {
        ServicioEnvio.mensajesEnviados.add("EMAIL: " + mensaje);
    }
}

File: python/gestorAplicacion/comunicacion/servicio_email.py
from gestoraplicacion.comunicacion.servicio_envio import ServicioEnvio


class ServicioEmail(ServicioEnvio):
    """
     Constructor asignando el mensaje al email.
     @param mensaje Mensaje a ser enviado.
    """
    def __init__(self, mensaje):
        super().__init__(mensaje)

    def enviar(self):
        super()._mensajes_enviados.append("EMAIL: " + super()._mensaje)